app/views/expense_items/_show.html.erb

```diff
@@ -0,0 +1 @@
+<li>Date: <%= expense_item.date.strftime("%d.%m.%Y") %> F: <%= expense_item.breakfast %>M: <%= expense_item.lunch %>A: <%= expense_item.dinner %>Description:<%= expense_item.annotation %></li>
```

Comment:
Gibt es einen Grund warum das hier alles in einer Zeile ist und das Datum immer auf das deutsche Format gestellt wird?